packages/components/select/README.md

```diff
@@ -35,17 +35,18 @@
 
 | Property | Attribute | Type | Default | Description |
 | --- | --- | --- | --- | --- |
-| dir | dir | String | ltr | Direction of the component 'rtl' or 'ltr'. |
+| dir | dir | String | ltr | Does component has uncommited changes or not. |
```

Comment:
What? 👀  Is it the readme script doing this? :D

Reply:
Yep, completely forgot to tell you this. For some reason it takes the description from the `dirty` property instead of `dir`. I did not investigate it, but this issue is reproducible if you will call `npm run update-doc-src-from-readme` and then `npm run update-readme`. You can investigate it in this branch if you have time and desire to do it :)